www/userguide/userguide.md

```diff
@@ -0,0 +1,290 @@
+* [Run history](#userguide-run-history)
+
+# <a name="userguide-user-guide"></a> Products
+The product system allows a single CodeChecker server to serve multiple separate
+result databases, named "products", under the same IP address and authentication
+domain.
+
+## <a name="userguide-managing-products"></a> Managing products
+![Products](images/products.png)
+
+After enabling the administrative actions by clicking on the
+*Show administration* button in the top right corner, click *Add new product*,
+then fill the form presented. The values that need to be filled here are the
+same as the arguments for `CodeChecker cmd products add`.
+These buttons are visible only for *Super Users*.
+
+![New product](images/new_product.png)
+
+If the product creation is successful, the window will disappear and the
+product will appear in the product list.
+
+Editing a product is done through the pencil icon, which is visible when
+administrative actions are enabled. This window lets you edit the product's
+configuration.
+
+![Edit product](images/edit_product.png)
+
+Products can be deleted by clicking on the red trash bin. This way the product
+is only unmounted from the server (losing access control data and connection),
+but **no analysis results are deleted**.
+
+![Remove product](images/remove_product.png)
+
+## <a name="userguide-managing-permissions"></a> Managing permissions
+* Server-wide permissions can be edited by clicking *Edit global permissions*.
+* Product-level permissions can be edited by clicking the edit icon for the
+product you want to configure the permissions.
+
+From the dropdown, select the permission you want to configure. The two lists
+show the users and groups known to the system - if a tick is present in its row,
+the given user or group has the permission directly granted. (Users who only
+have a certain permission through permission inheritance are not shown with a
+tick.)
+
+![Global permissions](images/product_global_permissions.png)
+
+Only the permissions you have rights to manage are shown in the dropdown.
+
+You can edit multiple permissions opening the window only once. Simply tick or
+untick the users/groups you want to give the permission to or revoke from them.
+Clicking *OK* will save the changes to the database.
+
+# <a name="userguide-list-of-runs"></a> List of runs
+List page contains the analysis runs available on the server under the selected
+product.
+
+You can do the following on this page:
+- [Filter runs](#userguide-filtering).
+- [Compare analysis](#userguide-compare-runs).
+- [Delete runs](#userguide-delete-runs).
+- [Sorting runs](#userguide-sorting-runs).
+
+![Runs](images/runs.png)
+
+## <a name="userguide-filter-runs"></a> Filter runs
+You can filter runs by run name using the input box above the run list table.
+The filter is working **case insensitive** and doing an **infix search**. If we
+start typing some keyword in this input box, the list are being _filtered
```

Comment:
Is it a *keyword*? Maybe some `phrase` or `name part`˛?

---

The list ~~are~~ is _automatically filtered_ as we type in.